src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -231,37 +249,57 @@ class DevTask extends AbstractFeatureTask {
         return result;
     }
 
-    private String dockerRunOpts;
+    private String containerRunOpts;
 
-    @Option(option = 'dockerRunOpts', description = 'Additional options for the docker run command when dev mode starts a container.')
+    @Option(option = 'containerRunOpts', description = 'Additional options for the container run command when dev mode starts a container.')
+    void setContainerRunOpts(String containerRunOpts) {
+        this.containerRunOpts = containerRunOpts;
+    }
+
+    private String dockerRunOpts;
+    @Option(option = 'dockerRunOpts', description = 'Alias for containerRunOpts')
     void setDockerRunOpts(String dockerRunOpts) {
-        this.dockerRunOpts = dockerRunOpts;
+        if (containerRunOpts == null) {
+            setContainerRunOpts(dockerRunOpts)
+        }
     }
 
-    private int dockerBuildTimeout;
+    private int containerBuildTimeout;
 
-    @Option(option = 'dockerBuildTimeout', description = 'Specifies the amount of time to wait (in seconds) for the completion of the Docker operation to build the image.')
-    void setDockerBuildTimeout(String inputValue) {
+    @Option(option = 'containerBuildTimeout', description = 'Specifies the amount of time to wait (in seconds) for the completion of the container operation to build the image.')
+    void setContainerBuildTimeout(String inputValue) {
         try {
-            this.dockerBuildTimeout = Integer.valueOf(inputValue);
+            this.containerBuildTimeout = Integer.valueOf(inputValue);
         } catch (NumberFormatException e) {
-            logger.error(String.format("Unexpected value: %s for dev mode option dockerBuildTimeout. dockerBuildTimeout should be a valid integer.", inputValue));
+            logger.error(String.format("Unexpected value: %s for dev mode option containerBuildTimeout. containerBuildTimeout should be a valid integer.", inputValue));
             throw e;
         }
     }
 
+    private int dockerBuildTimeout;
+    @Option(option = 'dockerBuildTimeout', description = 'Alias for containerBuildTimeout')
+    void setDockerBuildTimeout(String inputValue) {
+        setContainerBuildTimeout(inputValue)
```

Comment:
Why doesn't containerBuildTimeout take precedence here like you did for the other docker attributes?